bin/prettier.cjs

```diff
@@ -8,9 +8,11 @@ var packageJson = require("../package.json");
 pleaseUpgradeNode(packageJson);
 
 function runCli(cli) {
-  return cli.run(process.argv.slice(2));
+  return cli.run();
 }
 
 var dynamicImport = new Function("module", "return import(module)");
+var promise = dynamicImport("../src/cli/index.js").then(runCli);
 
-module.exports.promise = dynamicImport("../src/cli/index.js").then(runCli);
+// Exported for test
+module.exports.__promise = promise;
```

Comment:
Align with other debugging API?

```suggestion
module.exports.__test= { promise };
```

Reply:
I think it's unnecessary to wrap in a object, anyway we can change it anytime, so let's keep it this way.

BTW, someone don't know how to write ES5 JavaScript code 😄 .